scripts/src/create-workspace.js

```diff
@@ -103,37 +96,13 @@ async function main(argv) {
         initial: true,
       },
       {
-        type(_, { isPlugin }) {
-          if (isPlugin) {
-            return null;
-          }
-
-          return 'confirm';
-        },
+        type: 'confirm',
         name: 'isPublic',
         message: 'Is it a public package?',
         initial: true,
       },
```

Comment:
Not something that we need to change right away, but similar to react, I was wondering if there ever will be a case for something to be a private package here? Even in Compass monorepo I don't think we used this option much

Reply:
Yeah, that makes sense .. why wouldn't we make a package public here, we already have `scripts` anyway